### PR TITLE
Make Pending status mean eligible by policy

### PR DESCRIPTION
**Pull request by Mykhailenko**

### CODE REVIEW

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java [hunk 1]
```diff
@@ -154,7 +154,7 @@ private void makeTree(InternalJob job) {
         for (InternalTask td : job.getITasks()) {
             //if this task is a first task, put it in eligible tasks list
             EligibleTaskDescriptor lt = new EligibleTaskDescriptorImpl(td);
-
+            //            ((EligibleTaskDescriptorImpl) lt).getInternal().setStatus(TaskStatus.SUBMITTED);
```
Comment (chloegugli) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java]: Do you want to keep these comments

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java [hunk 2]
```diff
@@ -241,6 +241,7 @@ public void doLoop(TaskId initiator, Map<TaskId, InternalTask> tree, InternalTas
         for (Entry<TaskId, InternalTask> it : tree.entrySet()) {
             TaskId itId = it.getValue().getId();
             EligibleTaskDescriptorImpl td = new EligibleTaskDescriptorImpl(it.getValue());
+            //            td.getInternal().setStatus(TaskStatus.SUBMITTED);
```
Comment (chloegugli) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java]: Do you want to keep these comments?

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java [hunk 3]
```diff
@@ -420,6 +421,7 @@ public void doReplicate(TaskId initiator, Map<TaskId, InternalTask> tree, Intern
         for (Entry<TaskId, InternalTask> it : tree.entrySet()) {
             TaskId itId = it.getValue().getTaskInfo().getTaskId();
             EligibleTaskDescriptorImpl td = new EligibleTaskDescriptorImpl(it.getValue());
+            //            td.getInternal().setStatus(TaskStatus.SUBMITTED);
```
Comment (chloegugli) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java]: Do you want to keep these comments?

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java (hunk 1 above)
Comment (ShatalovYaroslav) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java]: can you remove commented code?

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java (hunk 2 above)
Comment (ShatalovYaroslav) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java]: can you remove commented code?

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java (hunk 3 above)
Comment (ShatalovYaroslav) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java]: can you remove commented code?

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerStateRecoverHelper.java [hunk 4]
```diff
@@ -278,8 +278,8 @@ private boolean runningTaskMustBeResetToPending(InternalTask task, RMProxy rmPro
     }
 
     private void setTaskStatusAndIncrementPendingCounter(TaskStatusCounter counter, InternalTask task) {
-        logger.info("Changing task status to " + TaskStatus.PENDING);
-        task.setStatus(TaskStatus.PENDING);
+        logger.info("Changing task status to " + TaskStatus.SUBMITTED);
```
Comment (fviale) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerStateRecoverHelper.java]: it seems weird to have this message at info level. It seems to be the case originally, but if we have 100K tasks, I can imagine the nicely polluted logs.

**Pull request merged**